src/view/Pages/Patients/PatientDetails.js

```diff
@@ -0,0 +1,230 @@
+    contactNumber: "",
+    nationality: "",
+  });
+
+  const patients = [
+    {
+      fullName: "Tom",
+      age: 23,
+      gender: "female",
+      dateOfBirth: "1999-05-20",
+      email: "tom@example.com",
+      contactNumber: "1234567890",
+      nationality: "INDIAN",
+    },
+  ];
+
+  const openEditModal = (patient) => {
+    setEditedPatient(patient);
+    setShowEditModal(true);
+  };
+
+  const closeEditModal = () => {
+    setShowEditModal(false);
+  };
+
+  const handleInputChange = (event) => {
+    const { name, value, type } = event.target;
+    const newValue = type === "radio" ? value : value.trim();
+    setEditedPatient((prevPatient) => ({
+      ...prevPatient,
+      [name]: newValue,
+    }));
+  };
+
+  const handleUpdatePatient = () => {
+    closeEditModal();
+  };
+
+  return (
+    <>
+      <Header />
+      <Sidebar />
+
+      <div className="centered-text">
+        <h2>Patients Details</h2>
+      </div>
+      {patients.map((patient, index) => (
+        <div className="cards-container" key={index}>
+          <div className="left-card">
+            <div className="flip-container">
+              <div className="flipper">
+                <div className="front">
+                  <div className="avatar">
+                    <div
+                      className={`avatar ${
+                        patient.gender === "male"
+                          ? "male-color"
+                          : "female-color"
+                      }`}
+                    >
+                      <FaUserAlt size={200} />
+                    </div>
+                  </div>
+                </div>
+                <div className="back">
+                  <h2>{patient.fullName}</h2>
+                  <p>Email: {patient.email}</p>
+                  <p>Contact: {patient.contactNumber}</p>
+                </div>
+              </div>
+            </div>
+          </div>
+
+          <div className="right-card">
+            <div className="main-text-style">
+              <p>Age </p>
+              <br />
+              <p>Date Of Birth </p>
+              <br />
+              <p>Gender </p>
+              <br />
+              <p>Address </p>
+              <br />
+              <p>Nationality </p>
+              <br />
+            </div>
+
+            <div className="sub-text-style">
+              <p> {patient.age}</p>
+              <br />
+              <p>{patient.dateOfBirth}</p>
+              <br />
+              <p>{patient.gender}</p>
+              <br />
+              <p>#23 building no.9 BGL</p>
+              <br />
+              <p>{patient.nationality}</p>
+              <br />
+            </div>
+
+            <div className="edit-button">
+              <FaEdit size={20} onClick={() => openEditModal(patient)} />
+              <br />
+              Edit
+            </div>
+          </div>
+        </div>
+      ))}
+
+      {showEditModal && (
+        <div className="edit-modal">
```

Comment:
move this edit modal in different component file so that in future we can reuse it for doctor edit modal